packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -229,10 +222,10 @@ function Cell(props) {
       ref={cellRef}
       aria-selected={isSelected}
       aria-label={ariaLabel}
-      tabIndex={isTabStop ? 0 : -1}
+      tabIndex={-1}
       className={className}
       {...(isRowHeader && { scope: 'row', role: 'rowheader' })}
-      onClick={onCellSelect ? onClick : undefined}
+      onMouseDown={onCellSelect ? onClick : undefined}
```

Comment:
I think the intentions are the same but are we sure we want to assign an `onClick` to an `onMouseDown`?

Reply:
Good catch.  I will update this.  I just missed making this update.